lib/api/api.js

```diff
@@ -44,7 +44,7 @@ auth.setHandler(vault);
 const api = {
     callApiMethod(apiMethod, request, log, callback, locationConstraint) {
         // no need to check auth on website or cors preflight requests
-        if (apiMethod === 'websiteGet' || apiMethod === 'websiteHead' ||
+        if (apiMethod === 'websiteHead' ||
```

Comment:
Thanks for your review @electrachong - I modified websiteHead + tests,  so it is compatible as well.